tests/Tests/500__jupyterhub/multiple-gpus.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Documentation    Tests for a scenario in which multiple GPUs are present in the cluster
+...              Specifically, we want to test for two nodes with 1 GPU each.
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/GPU.resource
+Resource         ../../Resources/Page/OCPDashboard/Pods/Pods.robot
+Library          JupyterLibrary
+Suite Setup      Spawner Suite Setup
+Suite Teardown   Double User Teardown
+Force Tags       JupyterHub
+
+
+*** Variables ***
+${NOTEBOOK_IMAGE} =         minimal-gpu
+
+
+*** Test Cases ***
+Verify Number Of Available GPUs Is Correct
+    [Documentation]  Verifies that the number of available GPUs in the
+    ...    Spawner dropdown is correct; i.e., it should show the maximum
+    ...    Number of GPUs available in a single node.
+    [Tags]    Tier1  Sanity  Resources-2GPUS
+    ...       ODS-1256
+    ${maxNo} =    Find Max Number Of GPUs In One Node
+    ${maxSpawner} =    Fetch Max Number Of GPUs In Spawner Page
+    Should Be Equal    ${maxSpawner}    ${maxNo}
+
+Verify Two Servers Can Be Spawned
+    [Documentation]    Spawns two servers requesting 1 gpu each, and checks
+    ...    that both can schedule and are scheduled on different nodes.
+    [Tags]    Tier1  Sanity  Resources-2GPUS
+    ...       ODS-1257
+    Spawn Notebook With Arguments  image=${NOTEBOOK_IMAGE}  size=Default  gpus=1
+    ${serial_first} =    Get GPU Serial Number
+    ${node_first} =    Get User Server Node
+    Close Browser
+    Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+    Login To RHODS Dashboard  ldap-user5  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
```

Comment:
could you use one of the test user in the test-variables file?

Reply:
I don't see any other users defined: https://github.com/red-hat-data-services/ods-ci/blob/master/test-variables.yml.example#L11

Reply:
hm ok, I haven't updated the test-variable example. Please refer to PR #208.